src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -1088,20 +1157,26 @@ int CpuemShim::xoclCreateBo(xclemulation::xocl_create_bo* info)
     ddr = 0;
   }
   
-  struct xclemulation::drm_xocl_bo *xobj = new xclemulation::drm_xocl_bo;
+  //struct xclemulation::drm_xocl_bo *xobj = new xclemulation::drm_xocl_bo;
+  auto xobj = std::make_unique<xclemulation::drm_xocl_bo>();
```

Comment:
Perfect!  This is what should be done in other places also.  :-)

Reply:
:)